### PR TITLE
jo: Add version 1.6

### PR DESCRIPTION
**Pull request by goostleek**
A tool for creating JSON objects from CLI

See https://github.com/jpmens/jo

### CODE REVIEW

File: bucket/jo.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/jpmens/jo/releases/download/1.6/jo.exe",
+            "hash": "ef0d182528adb77a4397f1f0700b356f644126d31599e39e375d40e136976327"
+        },
+        "64bit": {
+            "url": "https://github.com/jpmens/jo/releases/download/1.6/jo.exe",
+            "hash": "ef0d182528adb77a4397f1f0700b356f644126d31599e39e375d40e136976327"
+        }
+    },
```
Comment (rashil2000) [inline bucket/jo.json:15]:
```suggestion
    "url": "https://github.com/jpmens/jo/releases/download/1.6/jo.exe",
    "hash": "ef0d182528adb77a4397f1f0700b356f644126d31599e39e375d40e136976327",
```

File: bucket/jo.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://github.com/jpmens/jo/releases/download/$version/jo.exe"
+            },
+            "64bit": {
+                "url": "https://github.com/jpmens/jo/releases/download/$version/jo.exe"
+            }
+        }
```
Comment (rashil2000) [inline bucket/jo.json:26]:
```suggestion
        "url": "https://github.com/jpmens/jo/releases/download/$version/jo.exe"
```

File: bucket/jo.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.6",
+    "description": "A tool for creating JSON objects from CLImaking updates across multiple Github repositories with a single command",
```
Comment (rashil2000) [inline bucket/jo.json:3]:
```suggestion
    "description": "A tool for creating JSON objects from CLI",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Comment by goostleek** [comment]
Hi @rashil2000. Thx for the suggestions. I applied all of them.

**Pull request merged**